client/state/initial-state.js

```diff
@@ -252,13 +278,13 @@ async function getInitialStoredState( initialReducer ) {
 		}
 	}
 
-	await Promise.all( map( storageKeys, loadReducerState ) );
+	map( storageKeys, loadReducerState );
```

Comment:
maybe `forEach` instead of `map` since we don't need to build a return array?

Reply:
Good point, that was a silly oversight! Switched to a for loop for clarity.